evaluation/smatch_score.py

```diff
@@ -0,0 +1,58 @@
+import torch
+from enum import Enum
+
+
+class SmatchScore(Enum):
+    PRECISION = 1,
+    RECALL = 2,
+    F_SCORE = 3
+
+
+def initialize_smatch():
+    smatch = {
+        SmatchScore.PRECISION: 0.0,
+        SmatchScore.RECALL: 0.0,
+        SmatchScore.F_SCORE: 0.0
+    }
+    return smatch
+
+
+def to_pred_mat(mat: torch.tensor, sentence_len: int):
```

Comment:
maybe not the best naming (unpad matrix might have worked better)
Also, this function and the rest in this file could really use some comments cause it's hard to infer what the shapes of the input arguments could be